https://bugs.gentoo.org/444678

The damned ebuild leaves the working directory at
work/lib/ instead of work/

--- a/sys/sys/cdefs.h
+++ b/sys/sys/cdefs.h
@@ -228,7 +228,11 @@
 #if defined(__cplusplus) && __cplusplus >= 201103L
 #define	_Alignas(e)		alignas(e)
 #define	_Alignof(e)		alignof(e)
-#define	_Noreturn		[[noreturn]]
+#if __GNUC_PREREQ__(4, 8)
+#define _Noreturn		[[noreturn]]
+#else
+#define _Noreturn		__attribute__((__noreturn__))
+#endif
 #define	_Static_assert(e, s)	static_assert(e, s)
 /* FIXME: change this to thread_local when clang in base supports it */
 #define	_Thread_local		__thread
